src/vstart.sh

```diff
@@ -775,6 +775,7 @@ EOF
             ceph_adm -i "$key_fn" auth add osd.$osd osd "allow *" mon "allow profile osd" mgr "allow profile osd"
         fi
         echo start osd.$osd
+        sleep 1
```

Comment:
why?

Reply:
pidfile resource can't release quickly when crimson-osd quit from --mkfs, if not wait here, when start osd.0, it always report resource is busy, can't write to pidfile.